src/auth/auth.service.ts

```
@@ -6,6 +6,9 @@ import {
 } from '@nestjs/common';
 import { UserRole } from './enums/userRole.enum';
 import type { Repository } from 'typeorm';
+import type { User } from './entities/user.entity';
+import type { Portfolio } from './entities/portfolio.entity';
+import type { EmailToken } from './entities/email-token.entity';
 import { User } from './entities/user.entity';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)